docs/guide/groups.md

```diff
@@ -0,0 +1,47 @@
+Sometimes it can be necessary to create groups of repos, especially if the number
+of repos grows and if you have people in different teams work on different repositories.
```

Comment:
Mixing `repos` and `repositories`  in this sentence. Pick one and stick with it.